database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -1,9 +1,7 @@
 <?php
-
```

Comment:
do not remove the spaces here, it's purpose is for readability.

Reply:
I've fixed it back.